docs/docs/introducing-jsx.md

```diff
@@ -0,0 +1,202 @@
+---
+id: introducing-jsx
+title: Introducing JSX
+permalink: docs/introducing-jsx.html
+prev: hello-world.html
+next: rendering-elements.html
+---
+
+Consider this variable declaration:
+
+```js
+let element = <h1>Hello, world!</h1>;
+```
+
+This funny tag syntax is neither a string nor HTML.
+
+It is called JSX, and it is a syntax extension to JavaScript. We recommend using it with React to describe what the UI should look like. JSX may remind you of a template language, but it comes with a full power of JavaScript.
+
+>**Note:**
```

Comment:
Red does seem overly aggressive here. I can open a PR to fix this. If you need any other styling changes, send them my way.